configs/sot/siamese_rpn/README.md

```diff
@@ -30,11 +30,14 @@ We observe around 1.0 points fluctuations in Success and 1.5 points fluctuations
 
 The checkpoints from 10-th to 20-th epoch will be evaluated during training.
 
-After training, you need to pick up the best checkpoint from the log file, then use the best checkpoint to search the hyperparameters on UAV123 following [here](https://github.com/open-mmlab/mmtracking/blob/master/docs/useful_tools_scripts.md#siameserpn-test-time-parameter-search) to achieve the best results.
+If you want to get better results, you can pick up the best checkpoint from the log file, then use the best checkpoint to search the hyperparameters on UAV123 following [here](https://github.com/open-mmlab/mmtracking/blob/master/docs/useful_tools_scripts.md#siameserpn-test-time-parameter-search).
```

Comment:
`The checkpoints from 10-th to 20-th epoch will be evaluated during training. You can find the best checkpoint from the log file.`
`If you want to get better results, you can use the best checkpoint to search the hyperparameters on ...`